### PR TITLE
test

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: bids2openminds/main.py [hunk 1]
```diff
@@ -40,7 +39,7 @@ def create_approaches(layout_df):
     return list(approaches) or None
 
 
-def create_dataset_version(bids_layout, dataset_description, layout_df, studied_specimens, file_repository):
+def create_dataset_version(bids_layout, dataset_description, layout_df, studied_specimens, file_repository,collection):
```
Comment (lzehl) [inline bids2openminds/main.py:42]: space missing

File: bids2openminds/converter.py [hunk 2]
```diff
@@ -27,17 +30,19 @@ def convert(input_path, output_path=None, multiple_files=False, include_empty_pr
...
+    [files_list, file_repository] = main.create_file(layout_df, input_path,collection)
 
     dataset_version = main.create_dataset_version(
-        bids_layout, dataset_description, layout_df, subjects_list, file_repository
+        bids_layout, dataset_description, layout_df, subjects_list, file_repository,collection
     )
 
-    dataset = main.create_dataset(dataset_description, dataset_version)
+
+    dataset = main.create_dataset(dataset_description, dataset_version,collection)
```
Comment (lzehl) [inline bids2openminds/converter.py:43]: space missing

File: bids2openminds/converter.py [hunk 3]
```diff
@@ -27,17 +30,19 @@ def convert(input_path, output_path=None, multiple_files=False, include_empty_pr
 
     dataset_description = utility.read_json(dataset_description_path.iat[0, 0])
 
-    [subjects_dict, subject_state_dict, subjects_list] = main.create_subjects(subjects_id, layout_df, bids_layout)
+    [subjects_dict, subject_state_dict, subjects_list] = main.create_subjects(subjects_id, layout_df, bids_layout,collection)
```
Comment (lzehl) [inline bids2openminds/converter.py:33]: space missing

File: bids2openminds/converter.py [hunk 4]
```diff
@@ -27,17 +30,19 @@ def convert(input_path, output_path=None, multiple_files=False, include_empty_pr
...
+
 
-    [files_list, file_repository] = main.create_file(layout_df, input_path)
+    [files_list, file_repository] = main.create_file(layout_df, input_path,collection)
```
Comment (lzehl) [inline bids2openminds/converter.py:36]: space missing

File: bids2openminds/converter.py [hunk 5]
```diff
@@ -27,17 +30,19 @@ def convert(input_path, output_path=None, multiple_files=False, include_empty_pr
...
 
     dataset_version = main.create_dataset_version(
-        bids_layout, dataset_description, layout_df, subjects_list, file_repository
+        bids_layout, dataset_description, layout_df, subjects_list, file_repository,collection
```
Comment (lzehl) [inline bids2openminds/converter.py:39]: space missing

File: bids2openminds/converter.py [hunk 6]
```diff
@@ -46,7 +51,7 @@ def convert(input_path, output_path=None, multiple_files=False, include_empty_pr
         else:
             output_path = os.path.join(input_path, "openminds.jsonld")
 
-    globals.collection.save(output_path,individual_files=multiple_files,include_empty_properties=include_empty_properties)
+    collection.save(output_path,individual_files=multiple_files,include_empty_properties=include_empty_properties)
```
Comment (lzehl) [inline bids2openminds/converter.py:54]: spaces missing

File: bids2openminds/main.py [hunk 7]
```diff
@@ -96,12 +95,12 @@ def create_dataset_version(bids_layout, dataset_description, layout_df, studied_
...
     )
 
-    globals.collection.add(dataset_version)
+    collection.add(dataset_version)
 
     return dataset_version
 
 
-def create_dataset(dataset_description, dataset_version):
+def create_dataset(dataset_description, dataset_version,collection):
```
Comment (lzehl) [inline bids2openminds/main.py:103]: space missing

File: bids2openminds/main.py [hunk 8]
```diff
@@ -112,12 +111,12 @@ def create_dataset(dataset_description, dataset_version):
...
     )
 
-    globals.collection.add(dataset)
+    collection.add(dataset)
 
     return dataset
 
 
-def create_subjects(subject_id, layout_df, layout):
+def create_subjects(subject_id, layout_df, layout,collection):
```
Comment (lzehl) [inline bids2openminds/main.py:119]: space missing

File: bids2openminds/main.py [hunk 9]
```diff
@@ -214,16 +213,16 @@ def create_subjects(subject_id, layout_df, layout):
...
         subjects_dict[f"{subject}"] = subject_cache
         subjects_list.append(subject_cache)
-        globals.collection.add(subject_cache)
+        collection.add(subject_cache)
 
     return subjects_dict, subject_state_dict, subjects_list
 
 
-def create_file(layout_df, BIDS_path):
+def create_file(layout_df, BIDS_path,collection):
```
Comment (lzehl) [inline bids2openminds/main.py:221]: space missing

**Review by lzehl — CHANGES_REQUESTED** [review]
@Peyman-N can you make it PEP8 compliant (at least for the spaces?)

**Comment by Peyman-N** [comment]
This pull request has been split into two separate ones (#35 and #34)

**Pull request closed without merging**